citro3d/examples/triangle.rs

```diff
@@ -59,18 +59,24 @@ fn main() {
 
     let mut top_screen = gfx.top_screen.borrow_mut();
     let frame_buffer = top_screen.get_raw_framebuffer(Side::Left);
+    let (width, height) = (frame_buffer.width, frame_buffer.height);
 
     let mut instance = citro3d::Instance::new().expect("failed to initialize Citro3D");
 
-    let mut render_target = instance
-        .render_target_for_screen(
-            &frame_buffer,
-            ColorFormat::RGBA8,
-            DepthFormat::Depth24Stencil8,
-        )
+    let mut render_target = citro3d::render::Target::new(width, height, top_screen, None)
         .expect("failed to create render target");
 
-    render_target.set_output(&*top_screen, Side::Left);
+    render_target.set_output(Side::Left);
+    render_target.set_output(Side::Right);
```

Comment:
Just to know, what does this function actually do? Isn’y `select_rende_target` enough? Does it initialise the internal state or something?

Reply:
It does basically two things:
* Select which screen (top-left/top-right/bottom) to transfer pixels from the render target to the frame buffer, which I _believe_ happens when the frame ends
* For the purpose of the above, sets the input/output color format for the transfer to the screen's frame buffer

If you (for example) had two separate render targets for the left/right sides of the top screen, you'd call this to designate each one as a side.

Thinking about it more, though, it's not obvious this would need to be called for the bottom screen, so it might make sense to change the API to make it more straightforward to set up the transfer flags in that case. I'll see if I can come up with something reasonable, I guess.